op-e2e/actions/proofs/simple_program_test.go

```diff
@@ -0,0 +1,109 @@
+package proofs
```

Comment:
I think it would be good to separate the proofs tests from the helpers as well.  I'd forgotten that we were going to start putting tests into separate packages which I think is a great idea too, but then I think we should probably have `actions/proofs` for the tests and `actions/helpers/proofs` for the helper code.  That way if we wind up writing some tests for the helper code (good idea for many things - we have some for things like the l2_verifier), they'll be in the helper/proofs package and clearly testing the helper rather than being action tests in themselves.